providers/ms365/resources/ms365.lr

```diff
@@ -390,4 +393,30 @@ ms365.sharepointonline {
 ms365.teams {
   // CS Teams client configuration
   csTeamsClientConfiguration dict
+  // CS Teams Tenant Federated configuration
+  csTenantFederationConfiguration ms365.teams.tenantFederationConfig
+  csTeamsMeetingPolicy ms365.teams.teamsMeetingPolicyConfig
+}
+
+private ms365.teams.tenantFederationConfig {
+  identity string
+  allowedDomains dict                                                                                                                                               
+  blockedDomains dict
+  allowFederatedUsers bool
+  allowPublicUsers bool
+  allowTeamsConsumer bool
```

Comment:
This needs documentation. Sorry, I couldn't learn from the MS docs what this is.

Reply:
```suggestion
  // Whether to allow external Teams consumers 
  allowTeamsConsumer bool
```